configure.json

```diff
@@ -1,4 +1,29 @@
 {
+    "mss-check-hasredzone":{
+        "program": "mss-check-redzone",
+        "arguments": { "0": ["0"]},
+        "set-var": { "0": "has-redzone"}
```

Comment:
Do not set a variable. Let the results.json to record the result.
Therefore, the following getlen tests can depend on this one (test only if check-redzone returns 0)